src/main/resources/react4xp/_entries/Header.jsx

```diff
@@ -122,29 +125,32 @@ class Header extends React.Component {
           </div>
         </div>
         <Divider className="mobileMenuDivider" />
-        <div className={this.state.showMainMenuOnMobile ? 'showOnMobile header-content' : 'header-content'}>
-          <nav id="mainMenu" className="ssb-tabs">
-            {mainNavigation.map((topMenuItem, index) => {
-              const menuItemClick = this.toggleSubMenu.bind(this, index)
-              const activeMenuItem = this.state.indexForCurrentActiveMenuItem === index ||
+        <div className={this.state.showMainMenuOnMobile ? 'showOnMobile header-content' : 'hideOnMobile header-content'}>
+          <nav id="mainMenu" className="ssb-tabs" aria-label={mainMenuLabel}>
+            <ul className="tabItems">
+              {mainNavigation.map((topMenuItem, index) => {
+                const menuItemClick = this.toggleSubMenu.bind(this, index)
+                const activeMenuItem = this.state.indexForCurrentActiveMenuItem === index ||
                 (topMenuItem.isActive && this.state.indexForCurrentActiveMenuItem === undefined)
-              return (
-                <div key={index} className={`tabItem${activeMenuItem ? ' activeTab' : ''}`}>
-                  <button onClick={menuItemClick} >
-                    <span className={ activeMenuItem ? 'active navigation-item' : 'navigation-item'} >
-                      {activeMenuItem && this.state.showSubMenu ? <ChevronDown size="20" /> : <ChevronRight size="20" />}
-                      <span>{topMenuItem.title}</span>
-                    </span>
-                  </button>
-                  <Divider/>
-                  <ol className={this.state.showSubMenu ? 'visible subMenu' : 'subMenu' }>
-                    {this.renderSubMenu(topMenuItem, activeMenuItem)}
-                  </ol>
-                </div>
-              )
-            })}
+                return (
+                  <li key={index} className={`tabItem${activeMenuItem ? ' activeTab' : ''}`}>
+                    <button onClick={menuItemClick} aria-expanded={activeMenuItem ? 'true' : 'false' }>
```

Comment:
På denne måten blir de synlige og klikkbare menyvalgene som IKKE er i kolonnen man klikket på, ikke synlige for skjermleser selv om de er synlige visuelt. Er det meningen? Hierarkiet er igrunn logisk for skjermleser allikevel, så kanskje det er ok...

Reply:
Ja jeg var litt usikker på denne, og bare tok en avgjørelse. Jeg kan høre med Siv.